src/main/resources/application.properties

```diff
@@ -1,2 +1,7 @@
 spring.profiles.active=dev
-spring.mvc.hiddenmethod.filter.enabled=true
\ No newline at end of file
+spring.mvc.hiddenmethod.filter.enabled=true
+spring.mail.host=smtp.centrum.cz
+spring.mail.port=587
+spring.config.import = classpath:/email.properties

```

Comment:
This is causing issues with tests as this file doesnt exist on github.
Other than that, nice job! 👍